src/python/turicreate/toolkits/style_transfer/_tf_model_architecture.py

```diff
@@ -24,32 +24,37 @@ def define_tensorflow_variables(net_params, trainable=True):
     Parameters
     ----------
     trainable: boolean
-        If `true` the network updates the convolutional layers as well as the
-        instance norm layers of the network. If `false` only the instance norm
-        layers of the network are updated.
+        If `true` the transformer network updates the convolutional layers as
```

Comment:
In Python it's `True` and `False`, not `true` and `false`.

Reply:
Updated!